resource/js/make-plugin-callbacks.js

```diff
@@ -0,0 +1,29 @@
+const makeCallbacks = () => {
+  const newPage = window.SKOSMOS.pageType
+  const newUri = window.SKOSMOS.uri
+  const newPrefs = window.SKOSMOS.prefLabels
+  const ldJsonScript = document.querySelector('script[type="application/ld+json"]')
+  const embeddedJsonLd = ldJsonScript ? JSON.parse(ldJsonScript.innerHTML) : {}
+
+  const params = { uri: newUri, prefLabels: newPrefs, page: newPage, jsonLd: embeddedJsonLd }
```

Comment:
The `page` parameter name is a bit misleading. Now that we are changing the plugin API anyway, could we rename it to `pageType`?